gateway/1.10/modules/ROOT/pages/policies-included-xml-threat-protection.adoc

```diff
@@ -0,0 +1,102 @@
+= XML Threat Protection Policy
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+:keywords: xml threat protection, api gateway, flex gateway, gateway, policy 
+
+[width="100%", cols="1,3"]
+|===
+>s|Policy Name| XML Threat Protection
+>s|Summary| Protects against malicious XML in API requests
+>s|Category| Security
+>s|First Flex Gateway version available |v1.10.0
+>s|Returned Status Codes|400 - Bad Request
+|===
+
+== Summary
+
+Applications processing XML requests are susceptible to attacks characterized by unusual inflation of elements, attributes, and deep nesting levels. Attackers use recursive techniques to consume memory resources. Dramatic increases in the size of the application data often signal a security risk. Use the XML Threat Protection policy to reject unusual requests that exceed the defined expected size.
+
+== Configuring Policy Parameters
+
+include::partial$policy-title-headers.adoc[tag=configFile]
+
+----
+- policyRef:
+    name: xml-threat-protection
+  config:
+    maxNodeDepth: <number> // OPTIONAL, default: -1
+    maxAttributeCountPerElement: <number> // OPTIONAL, default: -1
+    maxChildCount: <number> // OPTIONAL, default: -1
+    maxTextLength: <number> // OPTIONAL, default: -1
+    maxAttributeLength: <number> // OPTIONAL, default: -1
+    maxCommentLength: <number> // OPTIONAL, default: -1
+----
+
+NOTE: Defining a parameter as `-1` means the parameter has no limit.
+
+[%header%autowidth.spread,cols="a,a,a,a"]
+|===
+|Parameter | Required or Optional | Default Value | Description
+
+| `maxNodeDepth`
+| Optional
+| -1
+| Specifies the maximum node depth of an XML document.
+
+| `maxAttributeCountPerElement`
+| Optional
+| -1
+| Specifies the maximum number of attributes in an element. Attributes used for defining namespaces are not counted.
+
+| `maxChildCount`
+| Optional
+| -1
+| Specifies the maximum number of children of an element in the XML document.
+
+| `maxTextLength`
+| Optional
+| -1
+| Specifies the maximum length in characters of text nodes in the XML document.
+
+| `maxAttributeLength`
+| Optional
+| -1
+| Specifies the maximum length in characters of an attribute in the XML document.
+
+| `maxCommentLength`
+| Optional
+| -1
+| Specifies the maximum number of comment characters in the XML document.
+
+|===
+
+==== Resource Configuration Example
+
+----
+- policyRef:
+    name: xml-threat-protection
+  config:
+    maxNodeDepth: 1
+    maxAttributeCountPerElement: 1
+    maxChildCount: 2
+    maxTextLength: 20
+    maxAttributeLength: 20
+    maxCommentLength: 20
+----
+
+include::partial$policy-title-headers.adoc[tag=ui]
+
+[%header%autowidth.spread,cols="a,a"]
+|===
+|Field |Description
```

Comment:
There is a move away from using tables for two-column [info](https://confluence.internal.salesforce.com/spaces/DOCTEAM/pages/143000889/tables+Style+Guidelines#:~:text=Reconsider%20tables%20that%20are%20one%20or%20two%20columns%20or%20rows.).

Reply:
We use this in all of our policies, so I'd rather reformat them as a whole then just this one.